### PR TITLE
Improve Github Documentation

### PR DESCRIPTION
**Pull request by KyomiDev**
**Please describe the changes this PR makes and why it should be merged:**
This PR will improve the documentation for discordo, by providing better information in the readme + contribution guide.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,4 +4,39 @@
...
+  <a href="#license">License</a>
+  •
+  <a href="#contributing">Contributing</a>
+  •
+  <a href="#credits">Credits</a>
+</p>
+
+## About
+Discordo is an easy to use Node.js wrapper for interacting with the Discord API.
+
```
Comment (PenPow) [inline README.md:25]: Can we add some more information to the about section, as it looks short compared to the rest of the documentation

File: .github/CONTRIBUTING.MD [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+If you want to help and contribute to the discordo.js project, then please read the section below. We use ESLint to help ensure code consistency, so please ensure you have set that up in your IDE prior to beginning work.
+
+## Setup
+
+To get ready to work on the codebase, please do the following:
+
+1. Fork & clone the repository.
+2. Run `npm ci`
+3. Write Code
+4. Run `npm run lint` to run ESLint and ensure any JSDoc changes are valid
```
Comment (PenPow) [inline .github/CONTRIBUTING.MD:11]: Add in the build step.

```shell
npm run build
npm run lint
```

**Review by PenPow — CHANGES_REQUESTED** [review]
Just a couple of small nits

File: .github/CONTRIBUTING.MD (hunk 2 above)
Comment (KyomiDev) [reply .github/CONTRIBUTING.MD]: Forgot about that 😅

Comment (KyomiDev) [reply .github/CONTRIBUTING.MD]: Fixed in 336a1b6

File: README.md [hunk 3]
```diff
@@ -4,4 +4,39 @@
     <img src="https://ss.penpow.dev/i/W4cYIr.png" width="546" alt="discordo" />
   </p>
   <br />
-</div>
\ No newline at end of file
+</div>
+
+<h3 align=center>An Easy to use Wrapper for the Discord API</a></h3>
```
Comment (PenPow) [inline README.md:9]: Fix Capitalisation on `Easy` and `Wrapper`

File: README.md [hunk 4]
```diff
@@ -4,4 +4,39 @@
...
+  •
+  <a href="#license">License</a>
+  •
+  <a href="#contributing">Contributing</a>
+  •
+  <a href="#credits">Credits</a>
+</p>
+
+## About
+Discordo is an easy to use Node.js wrapper for interacting with the Discord API. Aiming for fast releases with new discord features, and a robust, strongly typed codebase, providing first class typings for those using typescript, we aim to make developing a bot with our library as easy as possible.
```
Comment (PenPow) [inline README.md:24]: Decapitalize `Node`

File: README.md [hunk 5]
```diff
@@ -4,4 +4,39 @@
...
+
+## Installation
+To install discordo.js, you require Node.js V14.0.0 or newer, so that we can use the latest features.
```
Comment (PenPow) [inline README.md:27]: Ditto

**Review by PenPow — CHANGES_REQUESTED** [review]
Some more nits I found

**Comment by KyomiDev** [comment]
All of the above nits patched in 60789d5

File: README.md [hunk 6]
```diff
@@ -4,4 +4,39 @@
...
+  •
+  <a href="#license">License</a>
+  •
+  <a href="#contributing">Contributing</a>
+  •
+  <a href="#credits">Credits</a>
+</p>
+
+## About
+Discordo is an easy to use node.js wrapper for interacting with the Discord API. Aiming for fast releases with new discord features, and a robust, strongly typed codebase, providing first class typings for those using typescript, we aim to make developing a bot with our library as easy as possible.
```
Comment (PenPow) [inline README.md:24]: Capitalize Discord

Comment (PenPow) [inline README.md:24]: Capitalize Typescript

File: README.md [hunk 7]
```diff
@@ -4,4 +4,39 @@
...
+
+## Installation
+To install discordo.js, you require node.js V14.0.0 or newer, so that we can use the latest features.
```
Comment (PenPow) [inline README.md:27]: Decapitalize v14.0.0

**Review by PenPow — CHANGES_REQUESTED** [review]

**Comment by KyomiDev** [comment]
Nits again fixed in beaadcb

File: README.md [hunk 8]
```diff
@@ -4,4 +4,39 @@
...
+
+```Shell
+npm install discordo.js
+```
+
+## License
+Our code is released under the [Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0).
+
+## Contributing
+Before creating an issue/pull request, please ensure there is not an active one already created. If submitting a PR, please view the [contribution guide](https://github.com/discordo-js/discordo.js/blob/master/.github/CONTRIBUTING.md) before creating it.
```
Comment (PenPow) [inline README.md:37]: abbreviate pull request to PR

Comment (PenPow) [reply README.md:37]: Reason for this being done is because you abbreviate later, alternatively, you can remove the latter

**Review by PenPow — CHANGES_REQUESTED** [review]
Guess What! More incredibly minor nits

File: README.md (hunk 8 above)
Comment (KyomiDev) [reply README.md:37]: Will abbreviate the first for consistency with the rest of the docs

Comment (PenPow) [reply README.md:37]: 👍

**Comment by KyomiDev** [comment]
🎉

**Pull request merged**